### PR TITLE
[IE TESTS] Move Config behavior tests

### PR DESCRIPTION
**Pull request by antonzaycev96**

### CODE REVIEW

File: inference-engine/tests/functional/plugin/shared/CMakeLists.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@ addIeTarget(
 
 target_include_directories(${TARGET_NAME} PUBLIC
         ${PUBLIC_HEADERS_DIR}
+        ${IE_MAIN_SOURCE_DIR}/src/vpu/graph_transformer/include
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/CMakeLists.txt:32]: Why we add this lib? And Please move construction: `INCLUDE_DIRECTORIES` in addIeTarget

File: inference-engine/tests/functional/plugin/shared/src/behavior/config.cpp [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "targetDevice=" << targetDevice;
+    if (!configuration.empty()) {
+        result << "configItem=" << configuration.begin()->first << "_" << configuration.begin()->second;
+    }
+    return result.str();
+}
+
+void CorrectConfigTests::SetUp() {
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/behavior/config.cpp:37]: The number of tests is equal to the number of these tests in the old test infra?

**Comment by iefode** [comment]
In general, LGTM

File: inference-engine/tests/functional/plugin/shared/CMakeLists.txt (hunk 1 above)
Comment (antonzaycev96) [reply inference-engine/tests/functional/plugin/shared/CMakeLists.txt:32]: This is necessary to include the "vpu/private_plugin_config" .
Ok, I'll make a change.

File: inference-engine/tests/functional/plugin/shared/src/behavior/config.cpp (hunk 2 above)
Comment (antonzaycev96) [reply inference-engine/tests/functional/plugin/shared/src/behavior/config.cpp:37]: Yes

**Comment by azhogov** [comment]
ci/jenkins passed with internal checks

**Pull request merged**